### ISSUE TITLE
Overview diagram and surounding text don't match well

### ISSUE THREAD

**Issue by bachp** [telehash/telehash.org#171]
![image](https://cloud.githubusercontent.com/assets/333807/7012838/65892a88-dcb4-11e4-993f-af184e0441b8.png)

The drawing gives a very good overview over Telehash.
But in the surrounding text there are additional terms like channel, mesh and links used that are not shown in the diagram.
I think it would be advantageous to have all the mentioned terms appear in the graphic.

@temas What do you think?

**Comment by quartzjer** [comment]
I agree, but am bumping to 3.0.1 milestone (which should get a few other graphic updates too)

**Comment by temas** [comment]
Yeah, I've been debating if it's this diagram that changes or a really in-depth diagram that has absolutely every piece reference in the spec, good ol' full page dealy.
